content/svenska/submit.md

```diff
@@ -10,4 +10,22 @@ menu:
         pre: <i class="fas fa-paper-plane"></i>
 ---
 
-Kommer snart.
+Vetenskapsrådet och SciLifeLab har undertecknat Wellcome Trusts ställningstagande angående [Sharing research data and findings relevant to the novel coronavirus (COVID-19) outbreak](https://wellcome.ac.uk/coronavirus-covid-19/open-data), som bland annat anger att undertecknare säkerställer att forskningsresultat och data som har relevans för hanteringen av Coronaviruset snabbt delas och görs öppet tillgängliga:
```

Comment:
In order to have a capital letter here, I think it should be written as "Sharing research data and findings relevant to the novel coronavirus (COVID-19) outbreak" (with ") since Swedish is not as permitting for capital letters as English is.